### PR TITLE
Olexiy troschij

### PR DESCRIPTION
**Pull request by OlexiyTr**
<h1>Description</h1>

*Please include summary of the change in english*

<h2>How has this been tested?</h2>

- [ ] Executed code few times from start to end without crashing
- [ ] Only ran once
- [ ] Code have dedicated unit tests

### CODE REVIEW

File: tasks/task_3/transfer_learning.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+  tf.keras.layers.experimental.preprocessing.RandomFlip('horizontal'),
+  tf.keras.layers.experimental.preprocessing.RandomRotation(0.2),
+])
+
+preprocess_input = tf.keras.applications.mobilenet_v2.preprocess_input
+
+rescale = tf.keras.layers.experimental.preprocessing.Rescaling(1./127.5, offset= -1)
+
+IMG_SHAPE = IMG_SIZE + (3,)
+base_model = tf.keras.applications.MobileNetV2(input_shape=IMG_SHAPE,
```
Comment (meanalexrin) [inline tasks/task_3/transfer_learning.py]: If you are using mobilenet it's better to use 224x224 image shape

https://keras.io/api/applications/mobilenet/

But yeah, should work with 160, not an error, just may impact accuracy

**Review by meanalexrin — CHANGES_REQUESTED** [review]
Please move you work to the `/[username]_code` location, as described in readme
https://github.com/rnd4u-org/2021-knu-cairl#development 

Other than that - LGTM :+1:
